decentraland/unity-explorer
# fix: avatar alpha blending

## What does this PR change?

fixes #418 

This change fixes the last two avatar issues with alpha clipping that Alvaro found. I've added the 2 offending avatars into the Avatar Randomiser to make it easier to test.

## How to test the changes?

Load into game, load 30 test avatars from the Avatar Randomizer
Find the 2 avatars and check the alpha issues highlighted in the issue.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
